README.md

```diff
@@ -23,7 +23,7 @@ NATS:
   Endpoint: gnats://endpoint:4222
   RequestTimeout: 500ms # Default is 1s
 HTTP:
-  Port: 85 # Default is 80
+  Port: 8085 # Default is 8080
```

Comment:
I agree with 8085, but 80 is a default port reserved for HTTP communication, so that's OK

Reply:
Makes sence